_posts/2018-11-02-project-minimalistic-documentation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Project documentation 
+description: "Minimalistic but useful project documentation."
+author: bogdan
+modified: 2014-12-24
+tags: [documentation, agile, process, qa]
+categories: [Process]
+image:
+    feature: posts/project_documentation/title.jpg
+---
+
+We don’t like to write documentation. But we do like good documentation for tools we use. Which is a little bit controversial, isn’t it? Actually, this mean that we are too lazy to write docs, but we understand that it’s necessary for any good project. Here is few tips  how to create a “minimalistic but useful” documentation  from guy that don’t like to write documentation too. Hope it will be helpful.
+
+ <!-- more -->
+ 
+## THE PROBLEM
+It’s hard to determine level of detalization for docs. Too many details  and documentation will become outdated almost immediately. Too little detail - and it will be useless. So, what should be covered, and what is not? We can figure it out with a five right questions.
+
+## “WHAT”, “WHY”, “WHO”, “HOW” AND “HOW TO”
+### WHAT
+
+First of all, write down what is this. It should be clear and short - so anyone who will see it at first, will understand what is all about.  For example: *“Configuration.py is a library for configuration management in python apps.”*. 
+
+### WHY
+
+Next, what should be covered - it’s why this tool/project exists, and what problem you are trying to solve. How your team/company use it.  This is a good place to describe why this tool were created at the first place and what difference it has in compare to the alternatives (if some). This part is very important and often skipped, leaving beginners in perplexity and wondering why it exists, and the tool "X" not used instead. For example:
```

Comment:
Next thing to cover is why this tool/project exists, what problem you are trying to solve and how your team/company uses it.  This is a good place to describe why this tool was created in the first place and what the difference is compared to the alternatives (if there are any). This part is very important and often skipped, leaving beginners perplexed and bemused by its existence, and why they cannot use the tool "X" instead. For example: